File: examples/stream-sse.ts
import { VeroqClient } from '@veroq/sdk';

const client = new VeroqClient();
for await (const event of client.askStream("AAPL price and technicals")) {
  if (event.type === "data") console.log(`[${event.data.key}] loaded`);
  if (event.type === "summary_token") process.stdout.write(event.data.token);
  if (event.type === "done") console.log(`\nDone in ${event.data.response_time_ms}ms`);
}

File: examples/stream-sse.py
"""Stream financial intelligence in real-time via SSE."""
from veroq import VeroqClient

client = VeroqClient()
for event in client.ask_stream("What is AAPL stock price and technicals?"):
    if event["type"] == "thinking":
        print(f"Analyzing: {event['data'].get('intents', [])}")
    elif event["type"] == "data":
        print(f"[{event['data']['key']}] loaded")
    elif event["type"] == "summary_token":
        print(event["data"]["token"], end="", flush=True)
    elif event["type"] == "done":
        print(f"\n\nDone in {event['data']['response_time_ms']}ms")
